src/public/client.js

```diff
@@ -1,98 +1,101 @@
 "use strict";
 
 (function () {
-  
+
   /* -------- General Functions -------- */
-  
+
   /**
-  * Add an event to a DOM element
-  * @param {DOMElement} element - the element who listen
-  * @param {String} eventName - the name of the event
-  * @param {Function} callback - the function to execute
-  */
+   * Add an event to a DOM element
+   * @param {DOMElement} element - the element who listen
+   * @param {String} eventName - the name of the event
+   * @param {Function} callback - the function to execute
+   */
   function on(element, eventName, callback) {
     return element.addEventListener(eventName, callback);
   }
-  
+
   /**
-  * document.getElementById
-  * @param {String} id - the id of the DOMElement
-  * @return {DOMELement} 
-  */
+   * document.getElementById
+   * @param {String} id - the id of the DOMElement
+   * @return {DOMELement} 
+   */
   function getElementById(id) {
     return document.getElementById(id);
   }
 
   /**
-  * document.getElementById.value for input elements
-  * @param {String} id - the id of the DOMElement
-  * @return {String} 
-  */
+   * document.getElementById.value for input elements
+   * @param {String} id - the id of the DOMElement
+   * @return {String} 
+   */
   function getValueById(id) {
-    return document.getElementById(id).value;
+    return getElementById(id).value;
   }
-  
+
+  /// @TODO : review this function interest (used only once)
```

Comment:
and why review the interest of the function ? because it is used only once ?
That's not a reason to delete a function !